matrix/common/request/request_cache.py

```diff
@@ -64,6 +67,14 @@ def creation_date(self):
             self._creation_date = item[CacheTableField.CREATION_DATE.value]
         return self._creation_date
 
+    @property
+    def timeout(self) -> bool:
+        timeout = date.to_datetime(self.creation_date) > (date.get_datetime_now()) - timedelta(hours=1)
+        if timeout:
+            RequestTracker(self.retrieve_hash()).log_error("This request has timed out after 1 hour."
+                                                           "Please try again by resubmitting the POST request.")
```

Comment:
Would you need to update the driver for this to work?

Reply:
Nope, the driver as is will re-run the request if the RequestHash has an error